docker-compose-integration.yml

```
@@ -26,13 +27,13 @@ services:
     cap_add:
       - IPC_LOCK
     environment:
-      - VAULT_DEV_ROOT_TOKEN_ID=${VAULT_DEV_ROOT_TOKEN_ID}
-      - VAULT_ADDR=${VAULT_ADDR}
-      - VPC_PROXY=${VPC_PROXY}
+      - VAULT_DEV_ROOT_TOKEN_ID
+      - VAULT_ADDR=${VAULT_ADDR:-}
+      - VPC_PROXY=${VPC_PROXY:-}
```

this is the same as:
```suggestion
      - VAULT_ADDR
      - VPC_PROXY
```